operating/examine-images-for-vulnerabilities.adoc

```diff
@@ -0,0 +1,76 @@
+[id="examine-images-for-vulnerabilities"]
+= Examining images for vulnerabilities
+include::modules/common-attributes.adoc[]
+:context: examine-images-for-vulnerabilities
+
+toc::[]
+
+[role="_abstract"]
+With {product-title} you can analyze images for vulnerabilities.
+Scanner analyzes all image layers to check for known vulnerabilities by comparing them with the Common Vulnerabilities and Exposures (CVEs) list.
+
+When Scanner finds any vulnerabilities, it:
+
+* Shows them in the xref:../operating/manage-vulnerabilities.adoc#vulnerability-management-view_acs-operating-manage-vulnerabilities[*Vulnerability Management*] view for detailed analysis.
+* Ranks vulnerabilities according to risk and highlights them in the RHACS portal for risk assessment.
+* Checks them against enabled xref:../operating/manage-security-policies.adoc#manage-security-policies[security policies].
+
+Scanner inspects the images and identifies the installed components based on the files in the images.
+It may fail to identify installed components or vulnerabilities if the final images are modified to remove the following files:
+
+[cols=",a"options="header"]
+|===
+|Components |Files
+
+|Package managers
+|
+* `/etc/alpine-release`
+* `/etc/apt/sources.list`
+* `/etc/lsb-release`
+* `/etc/os-release` or `/usr/lib/os-release`
+* `/etc/oracle-release`, `/etc/centos-release`, `/etc/redhat-release`, or `/etc/system-release`
+* other similar system files.
+
+|Language-level dependencies
+|
+* `package.json` for JavaScript.
+* `dist-info` or `egg-info` for Python.
+* `MANIFEST.MF` in Java Archive (JAR) for Java.
+
+|Application-level dependencies
+|
+* `dotnet/shared/Microsoft.AspNetCore.App/`
+* `dotnet/shared/Microsoft.NETCore.App/`
+
+|===
+
+include::modules/scanning-images.adoc[leveloffset=+1]
+
+include::modules/pereodic-scanning-of-images.adoc[leveloffset=+1]
+
+include::modules/scan-inactive-images.adoc[leveloffset=+1]
+
+include::modules/fetching-vulnerability-definitions.adoc[leveloffset=+1]
+
+include::modules/understanding-vulnerability-scores.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../operating/examine-images-for-vulnerabilities.adoc#scan-inactive-images_examine-images-for-vulnerabilities[Scanning inactive images]
+* xref:../cli/getting-started-cli.adoc#getting-started-with-the-roxctl-cli[Getting started with the roxctl CLI]
+//Add link to integrate image scanners topic
+
+include::modules/view-images-in-your-environment.adoc[leveloffset=+1]
+
+include::modules/view-dockerfile-for-image.adoc[leveloffset=+1]
+
+include::modules/identify-container-image-layer-that-introduces-vulnerabilities.adoc[leveloffset=+1]
+
+include::modules/identify-operating-system-of-the-base-image.adoc[leveloffset=+1]
+
+include::modules/disable-language-specific-vulneribility-scanning.adoc[leveloffset=+1]
+
+[id="additional-resources"]
```

Comment:
You could update this anchor to the following, so that it is specific to this assembly:

`[id="additional-resources_examine-images-for-vulnerabilities"]`

Please see [this section of the OpenShift documentation guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies) for more details.